tool/templates/kotlin/Struct.kt.jinja

```diff
@@ -27,6 +28,55 @@ internal class {{type_name}}Native: Structure(), Structure.ByValue {
 
 {% endif -%}
 
+{% if !callback_params.is_empty() -%}
+    {%- for callback_param in callback_params %}
+internal interface Runner_{{callback_param.name}}: Callback {
+    fun invoke(ignored: Pointer?{% if callback_param.input_params_and_types != "" %}, {{callback_param.input_params_and_types}} {% endif %}): {{callback_param.output_type}}
+}
+
+class {{callback_param.name}}_Native: Structure(), Structure.ByValue {
+    @JvmField
+    internal var dataa: Pointer = Pointer(0L);
```

Comment:
Is this a typo?
```suggestion
    internal var data: Pointer = Pointer(0L);
```

Reply:
I found that `data` is a keyword in Kotlin, so I just added an "a" :sweat_smile: 
Perhaps I should rename it to some other word -- I was trying to preserve the same naming structure as the macro-side callback struct, that does have `data` as a field.

Reply:
ah! i'd maybe call it `data_`?

Reply:
oh, good idea! will do